conf/config-default.yaml

```diff
@@ -14,122 +14,121 @@
+  enable_reuseport: true       # If true, nginx SO_REUSEPORT option will be enabled.
+  show_upstream_status_in_response_header: false  # If true, include the upstream HTTP status code in
+                                                  # the response header `X-APISIX-Upstream-Status`.
+                                                  # If false, show `X-APISIX-Upstream-Status` only if
+                                                  # the upstream response code is 5xx.
   enable_ipv6: true
 
-  #proxy_protocol:                  # Proxy Protocol configuration
-  #  listen_http_port: 9181         # The port with proxy protocol for http, it differs from node_listen and admin_listen.
-                                    # This port can only receive http request with proxy protocol, but node_listen & admin_listen
-                                    # can only receive http request. If you enable proxy protocol, you must use this port to
-                                    # receive http request with proxy protocol
-  #  listen_https_port: 9182        # The port with proxy protocol for https
-  #  enable_tcp_pp: true            # Enable the proxy protocol for tcp proxy, it works for stream_proxy.tcp option
-  #  enable_tcp_pp_to_upstream: true # Enables the proxy protocol to the upstream server
-  enable_server_tokens: true        # Whether the APISIX version number should be shown in Server header.
-                                    # It's enabled by default.
-
-  # configurations to load third party code and/or override the builtin one.
-  extra_lua_path: ""                # extend lua_package_path to load third party code
-  extra_lua_cpath: ""               # extend lua_package_cpath to load third party code
-  #lua_module_hook: "my_project.my_hook"  # the hook module which will be used to inject third party code into APISIX
-
-  proxy_cache:                      # Proxy Caching configuration
-    cache_ttl: 10s                  # The default caching time in disk if the upstream does not specify the cache time
-    zones:                          # The parameters of a cache
-      - name: disk_cache_one        # The name of the cache, administrator can specify
-                                    # which cache to use by name in the admin api (disk|memory)
-        memory_size: 50m            # The size of shared memory, it's used to store the cache index for
-                                    # disk strategy, store cache content for memory strategy (disk|memory)
-        disk_size: 1G               # The size of disk, it's used to store the cache data (disk)
-        disk_path: /tmp/disk_cache_one  # The path to store the cache data (disk)
-        cache_levels: 1:2           # The hierarchy levels of a cache (disk)
-      #- name: disk_cache_two
+  # proxy_protocol:                    # PROXY Protocol configuration
+  #   listen_http_port: 9181           # APISIX listening port for HTTP traffic with PROXY protocol.
```

Comment:
i think the original comment is more clear?
```
# The port with proxy protocol for http, it differs from node_listen and admin_listen.
                                    # This port can only receive http request with proxy protocol, but node_listen & admin_listen
                                    # can only receive http request. If you enable proxy protocol, you must use this port to
                                    # receive http request with proxy protocol
```

Reply:
Original comment is assuming people don't understand PROXY protocol imho and its difference from other protocols (i.e. HTTP without PROXY protocol on L4). I kinda expect people looking for this config option understand this.

Reply:
ok, I can’t understand the original comment either.